src/unity.c

```diff
@@ -42,8 +42,11 @@ static const char UnityStrNaN[]                    = "NaN";
 static const char UnityStrDet[]                    = "Determinate";
 static const char UnityStrInvalidFloatTrait[]      = "Invalid Float Trait";
 #endif
+#ifndef UNITY_EXCLUDE_FILE
```

Comment:
I think thats wrong here?

```C

#ifndef UNITY_EXCLUDE_FILE
const char UnityStrErrFile[]                       = "Unity File Comparison Disabled";
#endif
```

Reply:
Since it's using memory compare to actually run the test, I'm not quite sure if the message will actually be used. I'll put in in there so if someone has a better idea of how it should be structured, it will be available.